### ISSUE TITLE
LeadBoard wont load

### ISSUE THREAD

**Issue by lorianla** [playgameservices/play-games-plugin-for-unity#1098]
Since i add this to my ScoreManager the LeaderBoard wont load.

Score Manager:
`
long currentScore = PlayerPrefs.GetInt ("HighScore");
		Social.ReportScore(currentScore, "CgkIs4WK5c8REAIQAQ", (bool success) => {
		// handle success or failure

			if(success){

				print ("successfully reported");

			}else{

				print("Something... wrong on report score");
			}
		} );
`

GooglePlayManager:

`
public void LeaderBoard(){
			if (PlayGamesPlatform.Instance.localUser.authenticated) {

				((PlayGamesPlatform)Social.Active).ShowLeaderboardUI(leaderboard_high_score);

			}
		}
`

**Comment by lorianla** [comment]
If i just sign in as a new player an press the leaderboard button it show's me the leaderboard. But if i play the game and the script report the highscore and i try then again to open the leaderboard it shows just the UI with the load icon. Any solution for this?

**Comment by lorianla** [comment]
@claywilkinson can you help me? :S

**Comment by claywilkinson** [comment]
Can you share a log of what is happening when you are submitting the score and when you try to show the leaderboard?

**Comment by lorianla** [comment]
@claywilkinson do you mean the log in the editor?

**Comment by claywilkinson** [comment]
No, on the device.  These APIs only work on a device.  For android you can use [logcat](http://developer.android.com/tools/debugging/debugging-log.html) to get the log.

**Comment by lorianla** [comment]
okay wait a moment search the log

**Comment by lorianla** [comment]
@claywilkinson okay since i addet this to the score manager 

		public void LeaderBoard(){
			if (PlayGamesPlatform.Instance.localUser.authenticated) {

				((PlayGamesPlatform)Social.Active).ShowLeaderboardUI(leaderboard_high_score);
				getHighScoreFromLeaderboard ();

			}
		}

	private void getHighScoreFromLeaderboard()
	{
		string[] UserFilter = { Social.localUser.id}; // Get the local player username
		HighScoreLeaderboard = PlayGamesPlatform.Instance.CreateLeaderboard(); // initiate 
		HighScoreLeaderboard.id = "CgkIs4WK5c8REAIQAQ"; // The string ID from Google console
		HighScoreLeaderboard.SetUserFilter(UserFilter); // Filter out other players
		HighScoreLeaderboard.LoadScores(RWResult =>
			{
				if(RWResult)
				{
					updateHighScoreOnDevice();
				}
			});
	}

	private void updateHighScoreOnDevice()
	{
		// Update MyHighScore variable with data pulled from the leaderboard.
		MyHighScore = (int) HighScoreLeaderboard.localUserScore.value;       
	}


it show's me the leadboard.

**Comment by claywilkinson** [comment]
Great! so everything is working?

**Comment by lorianla** [comment]
@claywilkinson i will test it out now with a ne gmail account. If everythink is working i close that issue. I tell you in a few minutes the result.

**Comment by lorianla** [comment]
@claywilkinson excuse me but is it possible that google play needs a little bit of time to 
update the leadboard when a new score is set?

**Comment by lorianla** [comment]
every time when i colide in the game scene with killcollider the script reports the score to the leaderboard and every time when the script reports the score to the leaderboard the leadboard just load and load

**Comment by claywilkinson** [comment]
How often are you submitting a score?  Your game may be busy processing the score submissions vs. doing other things.  Usually scores are submitted at the end of the game (or round) which makes your game much faster.

**Comment by lorianla** [comment]
If submit the score every 5 play's of a the scene that means if the scene was played 5 times it report's a score to the leaderboard. It's showing now the leaderboard but the score in the leaderboard show me just a score of "0"
